test_regress/t/t_trace_open_wrong_order.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2022 by Yu-Sheng Lin. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+if ($Self->{vlt_all}) {
+    compile(
+        verilator_flags2 => ["--cc --trace --exe $Self->{t_dir}/$Self->{name}.cpp"],
+        make_top_shell => 0,
+        make_main => 0,
+        );
+} else {
+    compile(
+        );
+}
+
+execute(
+    expect_filename => $Self->{golden_filename},
+    fails => 1
+    );
+
```

Comment:
```suggestion

file_grep($Self->{run_log_filename}, qr/trace\(\)' shall not be called after 'VerilatedVcdC::open\(\)'/i);

```

So then your .out won't be sensitive to the line number.